index.d.ts

```diff
@@ -155,6 +155,7 @@ declare module '@reportportal/client-javascript' {
     launchUuidPrintOutput?: string;
     restClientConfig?: RestClientConfig;
     token?: string;
+    skippedIssue?: boolean;
```

Comment:
I found that we have [skippedIsNotIssue](https://reportportal.io/docs/log-data-in-reportportal/HowToReportAttributesToReportPortal/#skippedisnotissue) naming for launch imports.
Let's rename this prop to be consistent.